packages/package_info/android/gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.jvmargs=-Xmx1536M
+android.enableJetifier=true
```

Comment:
Based on the name, I'm hoping that `enableJetifier` does not imply `enableAndroidX`, so it should reduce compilation errors without causing compilation errors.

Reply:
I don't think this has any meaning without `enableAndroidX`, and if it does I think it would also implicitly turn AndroidX on. If AndroidX is enabled and this flag is true, Gradle will attempt to run Jetifier to convert any support references to AndroidX ones. I'm not sure how Gradle handles just this flag with `enableAndroidX` being undefined.

I'm pretty sure that putting this here means that this subproject and only this subproject would be affected, but there may be some weirdness here on account of our build system.